pkgs/swiftgen/swift2objc/Sources/swift2objc/AST/ParameterSyntax.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  File.swift
+//  
+//
+//  Created by Mohammad Eid on 13/05/2024.
+//
+
+import Foundation
+
+struct ParameterSyntax {
+    let label: String?
```

Comment:
There are two fields for the parameter name cause a swift function parameter can have an inner and outer name. `label` was supposed to hold the outer name (which can be nil in case of a positional parameter) and `name` was supposed to hold the inner one. My naming there was quite bad there sorry XD.
 
I improved the naming and added a comment to make things clearer.